charts/hivemq-platform/values.yaml

```diff
@@ -134,7 +153,8 @@ license:
   dataHub: {}
     ## Defines the name of the HiveMQ Data Hub license in the Kubernetes Secret (the .plic suffix will be added automatically).
     #data-hub-license-name:
-    #  # Inlines the HiveMQ Data Hub license as an encoded 64-bytes string.
+    #  # Inlines the HiveMQ Data Hub license as a string.
+    #  # By default, this data must be a Base64 encoded string. Otherwise, set isLicenseBase64Encoded=false to use a clear string.
```

Comment:
By default, this data is expected as Base64 encoded string. Set "isLicenseBase64Encoded=false" to provide a plain string.